examples/rust/standalone/qdrant-example/src/main.rs

```diff
@@ -0,0 +1,190 @@
+use qdrant_client::prelude::*;
+use qdrant_client::qdrant::vectors_config::Config;
+use qdrant_client::qdrant::{CreateCollection, SearchPoints, VectorParams, VectorsConfig};
+use serde_json::{self, json};
+
+const ENCRYPTED_DOC_KEY: &str = "_encrypted_document";
+
+#[tokio::main]
+async fn main() -> Result<()> {
+    // Example of top level client
+    let client = QdrantClient::from_url("http://localhost:6334").build()?;
+    let base64_engine = base64::engine::general_purpose::STANDARD_NO_PAD;
+    let alloy_client = create_alloy_client()?;
+    let tenant_name = "tenant1";
+    let tenant_id = TenantId(tenant_name.to_string());
+    let alloy_metadata = AlloyMetadata::new_simple(tenant_id);
+    client.delete_collection(tenant_name).await?;
+
+    client
+        .create_collection(&CreateCollection {
+            collection_name: tenant_name.into(),
+            vectors_config: Some(VectorsConfig {
+                config: Some(Config::Params(VectorParams {
+                    size: 10,
+                    distance: Distance::Cosine.into(),
+                    ..Default::default()
+                })),
+            }),
+            ..Default::default()
+        })
+        .await?;
+
+    let json = json!(
+        {
+            "foo": "Bar",
+            "bar": 12,
+            "baz": {
+                "qux": "quux"
+            }
+        }
+    );
+
+    // Since the embedding is sensitive, we'll encrypt the text as well. We can encrypt it as a single blob since we don't
+    // need to match on pieces for this example. If we wanted to match on some fields we could instead deterministically encrypt those fields.
+    let encrypted_json = alloy_client
+        .standard_attached()
+        .encrypt(
+            serde_json::to_string(&json).unwrap().as_bytes().to_vec(),
+            &alloy_metadata,
+        )
+        .await?;
+
+    // Encrypt the embedding as well.
+    let encrypted_vector = alloy_client
+        .vector()
+        .encrypt(create_plaintext_vector(vec![12.; 10]), &alloy_metadata)
+        .await?;
+
+    let payload: Payload = {
+        let mut inner = Payload::new();
+        inner.insert(ENCRYPTED_DOC_KEY, base64_engine.encode(encrypted_json.0));
+        inner
+    };
+
+    let points = vec![PointStruct::new(
+        0,
+        encrypted_vector.encrypted_vector,
+        payload,
+    )];
+    client
+        .upsert_points_blocking(tenant_name, None, points, None)
+        .await?;
+
+    // Later we want to query, but because the embeddings are encrypted, we need to encrypt the query.
+    let query_vector = alloy_client
+        .vector()
+        .encrypt(create_plaintext_vector(vec![11.; 10]), &alloy_metadata)
+        .await?;
+    let search_result = client
+        .search_points(&SearchPoints {
+            collection_name: tenant_name.into(),
+            vector: query_vector.encrypted_vector,
+            filter: None,
```

Comment:
Their example shows a filter, so might be worth doing the deterministic encryption to show that still works